include/wx/qt/radiobut.h

```diff
@@ -32,15 +33,21 @@ class WXDLLIMPEXP_CORE wxRadioButton : public wxControl
                  const wxValidator& validator = wxDefaultValidator,
                  const wxString& name = wxRadioButtonNameStr );
 
+    virtual ~wxRadioButton();
+
     virtual void SetValue(bool value);
     virtual bool GetValue() const;
 
     virtual QWidget *GetHandle() const;
 
-protected:
+    void addButtonToGroup(QRadioButton* radioButton);
 
 private:
+    void createAndJoinNewGroup( wxWindow *parent );
```

Comment:
Function names should start with capitals.

Reply:
Noted.